libs/wingsdk/src/std/file.ts

```diff
@@ -0,0 +1,34 @@
+import * as pathext from "path";
+import { Code, InflightClient } from "../core";
+
+
+export class File {
+
+  /**
+   * @internal
+  */
+  public static _toInflightType(): Code {
+    return InflightClient.forType(__filename, this.name);
+  }
+
+  public static getExtension(fileName: string): string {
+    return pathext.extname(fileName).toLowerCase();
+  }
+
+  public static getFileEncoding(fileName: string): "base64" | "utf8" {
+    const type = this.getExtension(fileName);
+    switch (type) {
```

Comment:
I think it's better to let the user specify the encoding of the file, and set the default to a common one (utf-8 for example) as there are probably dozens of "utf-8" compatible files that aren't in this list, as well as cases which the user prefers to go with a completely other encoding
(it's ok to add an extra argument to the `addFile` signature 🙂 ) 
What do you think?

Reply:
Yeah, that would be great. Let me add an extra argument file encoding.